src/olympia/bandwagon/admin.py

```diff
@@ -87,5 +96,14 @@ def has_add_permission(self, request):
             request.user, amo.permissions.ADMIN_CURATION
         ) or super().has_add_permission(request)
 
+    def delete_model(self, request, obj):
```

Comment:
Is there a reason we don't want a deleted log in all cases? Why only via admin?

Reply:
I don't think there is a strong reason, it's just something that never came up (where as deletion by moderators is something we need to keep track of for the transparency report - we already had the `LogEntry` django generates automatically for actions happening in the admin, but that's not as centralized as our `ActivityLog`).

Reply:
FWIW ratings behave in the same way, the only activity log we emit is for moderators/reviewers deletion